### DISCUSSION TITLE
PTZ possible with wz_mini?

### DISCUSSION THREAD

**Discussion by woodyl** [gtxaspec/wz_mini_hacks#448] [Q&A]
Are any PTZ functions available using wz_mini? I see a reference to a broken tilt function in the notes, but I don't see any documentation of PTZ anywhere else.

**Comment by bjs-pdx** [comment]
I can confirm on a Pan V2 horizontal (pan) motion works,  **vertical (tilt) does not.**  See note below.

Missed this on the main page and haven't tested as I no longer have wz_mini installed on my Pan V2:



    Everything works except:
        PAN v2:
            Tilt (Vertical) only works at motor speed 9

Update:  Set the motor speed to 9 on my Pan V2 and vertical tilt works, sort of.  I still get the "reached the end" message and it looks like the lens hasn't moved, at least until I reboot the camera and find that the tilt has, in fact, changed.

**Reply by sibero80** [reply]
Can you provide details on how you send motor commands to the camera?

**Reply by bjs-pdx** [reply]
Just via the Wyze app. I’m probably going to give up on the Pan Cam. Just doesn’t work completely.
